### PR TITLE
Fixing strip_invalid return value order

### PR DESCRIPTION
**Pull request by rafaelmagu**
See issue #3.

Also commented out some debugging output that was causing errors in the post_data_to_server.

### CODE REVIEW

File: lib/rumeme/sms_interface.rb [hunk 1]
```diff
@@ -170,22 +170,22 @@ def create_login_string # can be calculate once at initialization
...
 
     def post_data_to_server data
-      puts 'post_data_to_server'
+      # puts 'post_data_to_server'
 
       http_connection = open_server_connection(@server_list[0])
       text_buffer = create_login_string + data
 
-      puts "buffer: #{text_buffer}"
+      # puts "buffer: #{text_buffer}"
```
Comment (rafaelmagu) [inline lib/rumeme/sms_interface.rb:178]: I don't think it's a good idea to puts the request out as it contains the username/password in plain-text.

File: lib/rumeme/sms_interface.rb [hunk 2]
```diff
@@ -170,22 +170,22 @@ def create_login_string # can be calculate once at initialization
...
 
       resp = http_connection.post(path, text_buffer, headers)
       data = resp.body
-      p resp
-      p data
+      # p resp
+      # p data
       
-      raise BadServerResponse.new('http response code != 200') unless response.code.to_i == 200
+      raise BadServerResponse.new('http response code != 200') unless resp.code.to_i == 200
```
Comment (rafaelmagu) [inline lib/rumeme/sms_interface.rb:188]: response.code.to_i was raising a NameError. Assumed dev wanted to use resp.code.to_i.

File: lib/rumeme/sms_interface.rb [hunk 3]
```diff
@@ -200,8 +200,8 @@ def post_data_to_server data
       response_message.match /^(\d+)\s+/
       response_code = $1.to_i
 
-      puts "latest response code: #{response_code}"
-      puts "response: #{response_message }"
+      # puts "latest response code: #{response_code}"
+      # puts "response: #{response_message }"
```
Comment (rafaelmagu) [inline lib/rumeme/sms_interface.rb:204]: No need to output anything if sending was successful. Raise exceptions otherwise.

**Comment by antlypls** [comment]
@rafaelmagu Thanks for contribution.

File: lib/rumeme/sms_interface.rb (hunk 1 above)
Comment (antlypls) [inline lib/rumeme/sms_interface.rb:178]: It was used for debugging of the first version.
The better logging solution should be used in this case, I'll remove console logging.

**Pull request merged**